katz/src/main/kotlin/katz/arrow/FunctionK.kt

```diff
@@ -16,7 +16,7 @@
 
 package katz
 
-interface FunctionK<F, G> {
+interface FunctionK<in F, out G> {
```

Comment:
could be, but compiler warn me about they could have in/out variante in order to be more precise using their types so i add them.

Could let's keep with in/out that at this point don't hurt and remove a lint warning ,we added deteck, but need could need also a lint (if not implode with our generics 🤣 )